### ISSUE TITLE
Competitive tweeting feature

### ISSUE THREAD

**Issue by ghost** [twitter/the-algorithm#2081]
**Describe the bug**
Why is there no ranked competitive tweeting like there is for many other online games where people also spam gamer words in chat? It would be a huge QoL improvement for Twitter dot com to get with the times and add ranked competitive matchmaking.

**To Reproduce**

**Expected behavior**
Competitive, ranked twitter matchmaking

**Screenshots**

**Environment**
I use arch btw

**Additional context**
Arch is better than windows you should try it

**Comment by dumbButSkilledDev** [comment]
WE NEED THIS

**Comment by Palkiaa** [comment]
yeah, i'd play Twitter often if it wasn't just another battle royale with kids tweet-spamming
